cosmos/x/evm/plugins/txpool/mempool/mempool.go

```diff
@@ -116,20 +116,20 @@ func (etp *EthTxPool) GetTransactions() coretypes.Transactions {
 
 // GetNonce returns the nonce for the given address from the mempool if the address has sent a tx
 // in the mempool.
-func (etp *EthTxPool) GetNonce(addr common.Address) (uint64, bool) {
+func (etp *EthTxPool) GetNonce(addr common.Address) (uint64, error) {
```

Comment:
I think this might be a regression, since you're going to be returning an error if the nonce is not found, which should just mean nonce 0

Reply:
true it should be that simple